### DISCUSSION TITLE
Seeking help for design of dashboards for Carbon Emission tracking project.

### DISCUSSION THREAD

**Discussion by PKYadav2023** [getredash/redash#6376] [Q&A]
Your contribution can be volunteer or paid we are eager to hear how you can help us. 

We are actively seeking proficient individuals well-versed in Redash (redash.io) software, and adept data science experts who are enthusiastic about contributing to a transformative endeavour. Our project aims to cultivate a culture of carbon neutrality and eco-conscious living among individuals. Central to this endeavour is the development of comprehensive dashboards that resonate with our mission.

As part of our initiative, we gather data from diverse partners and diligently transform it into structured formats. Leveraging this meticulously structured data, we have undertaken the creation of an array of dashboards employing specialised queries. We are in need of dedicated volunteers who can offer their expertise in the following capacities:

*Data Science:*

1. *Data Table Enhancement:* Conduct an in-depth review of our structured data tables in relation to the raw data. Your insights will aid in refining these tables, not only to economize on resources but also to expedite the generation of insightful analyses.

2. *Data Pipeline Optimisation:* We seek your guidance to enhance the efficiency of our data pipelines holistically. Your expertise will play a pivotal role in streamlining our data processing mechanisms.

*Redash:*

3. *Query Refinement:* Although preliminary queries have been established, their outcomes in Redash have fallen short of our expectations. We require you to rectify and optimize these queries, thereby aligning the results with our intended objectives.

4. *Graphical Visualization Enhancement:* Our current widgets, composed of charts and graphs derived from queries, fail to capture the vibrancy displayed on the Redash platform. Your creative prowess will be invaluable in crafting captivating and compelling graphical representations. These visuals will not only energise the dashboards but also engage users in a manner that conveys the essence of their carbon footprint in diverse and impactful ways.

If you possess the technical acumen and enthusiasm to contribute to this significant cause, we invite you to join our team of volunteers. By dedicating your expertise, you will actively participate in fostering a sustainable future. Your commitment will serve as a beacon, illuminating the path toward a greener and more environmentally conscious society.

**Comment by justinclift** [comment]
> we invite you to join our team of volunteers.

Is there a website, or other "further info" type of thing people can take a look at? :smile:
